### PR TITLE
ref(getting-started-docs): Clean up platforms logic

### LINKED ISSUE #54670: Don't rely on sentry-docs to display platforms in the onboarding

### PR DESCRIPTION
**Pull request by priscilawebdev**
This PR clean up the file `static/app/data/platforms.tsx` removing the need to use `
import integrationDocsPlatforms from 'integration-docs-platforms';` and filters

Closes: https://github.com/getsentry/sentry/issues/54670

### CODE REVIEW

File: static/app/data/platforms.tsx [hunk 1]
```diff
@@ -1,108 +1,689 @@
-import integrationDocsPlatforms from 'integration-docs-platforms';
```
Comment (evanpurkhiser) [inline static/app/data/platforms.tsx:1]: Let's make sure we don't forget to cleanup all the things that handle downloading this.

* Webpack configuration that sets up this `integration-docs-platforms` alias
* Webpack plugin to download
* Python implementation to download

Comment (matejminar) [reply static/app/data/platforms.tsx:1]: @priscilawebdev is it this issue? https://github.com/getsentry/sentry/issues/53861
I also created a task for us to not forget about @evanpurkhiser 's items.
https://github.com/getsentry/sentry/issues/54835

Comment (priscilawebdev) [reply static/app/data/platforms.tsx:1]: yes @matejminar before we can completely remove everything related to the `sentry-docs` fetch, we have to migrate the wizard docs for profiling and replay

**Comment by priscilawebdev** [comment]
@ArthurKnaus passing this one to you too 😉

**Review by evanpurkhiser — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
Let's just make sure we follow up on the items I listed in my earlier comment!

**Pull request merged**